### PR TITLE
add support to skip boolean conversion for specified meta keys

### PR DESCRIPTION
**Pull request by klevron1**

### CODE REVIEW

File: consul_awx.py [hunk 1]
```diff
@@ -83,10 +85,17 @@ def build_full_inventory(self, node_meta=None, tagged_address="lan"):
...
                     continue
 
-                try:
-                    value = str2bool(value.strip())
-                except ValueError:
-                    pass
+                # Keep from converting some values to boolean
+                # So a valid value of 1 or 0 is kept as is
+                convert_to_bool = True
+                if node_meta_types and key in node_meta_types:
```
Comment (wilfriedroset) [inline consul_awx.py]: `convert_to_bool` is used only one time below.
I reckon we can change a little bit the if statement to not need it.
Something like
```python
if not (node_meta_types and key in node_meta_types):
    try:
        value = str2bool(value.strip())
    except ValueError:
        pass
```

File: consul_awx_test.py [hunk 2]
```diff
@@ -111,6 +129,12 @@ def test_mock(mocked_catalog_nodes, mocked_catalog_node):
                         "server_type": "postgresql",
                         "cluster": "one-two",
                     },
+                    "node5": {
+                        "ansible_host": "{}.0.0.4".format(ip_prefix),
+                        "datacenter": "dc1",
+                        "server_type": "postgresql",
+                        "cluster": 1,
```
Comment (wilfriedroset) [inline consul_awx_test.py:136]: should this be a string?

File: consul_awx_test.py [hunk 3]
```diff
@@ -174,3 +221,13 @@ def test_get_node_meta_configfile():
         print(fp.name)
         path = fp.name
         assert get_node_meta(path) == {"k1": "v1"}
+
+
+def test_get_node_meta_types_configfile():
+    with tempfile.NamedTemporaryFile() as fp:
+        fp.write(b"[consul_node_meta_types]\ncluster:skip_bool")
+        fp.seek(0)
+        print(dir(fp))
```
Comment (wilfriedroset) [inline consul_awx_test.py]: do we still need the `print`?

**Comment by wilfriedroset** [comment]
Thank you for the PR, I've left a couple a comments let me know what you think about them.

File: consul_awx_test.py (hunk 2 above)
Comment (klevron1) [reply consul_awx_test.py:136]: That value is formatted as an integer in get_host_vars
https://github.com/wilfriedroset/consul-awx/blob/master/consul_awx.py#L166-L167

File: consul_awx_test.py (hunk 3 above)
Comment (klevron1) [reply consul_awx_test.py]: I just copied from the test_get_node_meta_configfile and just changed the names.
So not sure why the print is there, but I have removed it from both.

**Pull request merged**